core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.ts

```diff
@@ -53,15 +53,14 @@ import { FIELD_TYPES } from '../dot-edit-content-field/utils';
     ]
 })
 export class DotEditContentFormComponent implements OnInit {
-    @Input() formData!: EditContentPayload;
+    @Input()
+    formData!: EditContentPayload;
     @Output() changeValue = new EventEmitter();
-
+    form!: FormGroup;
+    protected tabs: DotCMSContentTypeLayoutTab[] = [];
```

Comment:
@oidacra I have a doubt, why use protected?

Reply:
This file was touched by the lint. No idea.